src/modules.js

```diff
@@ -344,6 +344,29 @@ function isExportedByForceFilesystem(name) {
          name === 'removeRunDependency';
 }
 
+// When running with ASSERTIONS enabled we create stubs for each library
+// function that that was not included in the build.  This gives usefull errors
+// when library dependencies are missing from `__deps` or depended on without
+// being added to DEFAULT_LIBRARY_FUNCS_TO_INCLUDE
+// TODO(sbc): These errors could potentially be generated at build time via
+// some kind of acron pass that searched for uses of these missing symbols.
+function addMissingLibraryStubs() {
+  if (!ASSERTIONS) return '';
+  let rtn = '';
+  const librarySymbolSet = new Set(libraryFunctions);
+  for (const ident in LibraryManager.library) {
+    if (typeof LibraryManager.library[ident] === 'function') {
+      if (ident[0] === '$' && !isJsLibraryConfigIdentifier(ident) && !LibraryManager.library[ident + '__internal']) {
```

Comment:
```suggestion
      if (ident[0] === '$' && !isJsLibraryConfigIdentifier(ident)) {
```
`__internal` is handled in that helper function

Reply:
This is checking if there exists another entry called `ident + '__internal'`..  isJsLibraryConfigIdentifier only checks `ident` itself.

Reply:
I created a helper to make this more clear.